working_with_fhir/working_with_fhir_parsing.bal

```diff
@@ -0,0 +1,25 @@
+import ballerinax/health.fhir.r4.parser as parser;
```

Comment:
Any advantages of having this alias? The prefix is anyway `parser`.

Reply:
Typo.